package.json

```diff
@@ -30,26 +30,26 @@
     "eslint": ">=0.8.0"
   },
   "devDependencies": {
-    "@types/eslint": "^7.2.0",
-    "@types/jest": "^26.0.0",
-    "@types/node": "^14.0.13",
-    "@typescript-eslint/eslint-plugin": "^3.4.0",
-    "@typescript-eslint/parser": "^3.4.0",
-    "eslint": "^7.3.1",
-    "eslint-config-airbnb-base": "^14.2.0",
-    "eslint-config-prettier": "^6.11.0",
-    "eslint-import-resolver-typescript": "^2.0.0",
-    "eslint-plugin-import": "^2.21.2",
-    "eslint-plugin-jest": "^23.17.1",
+    "@types/eslint": "^7.28.2",
+    "@types/jest": "^27.0.2",
+    "@types/node": "^16.11.7",
+    "@typescript-eslint/eslint-plugin": "^5.3.1",
+    "@typescript-eslint/parser": "^5.3.1",
+    "eslint": "^8.2.0",
```

Comment:
Sticking to ESLint v7 in the dev dependencies would unblock this PR could (not really sure where the compatibility with ESLint v8 is added/tested in this PR repo). Upgrading ESLint here is just for the local environment used to develop the project, that must not block this PR to add ESLint v8 compatibility to the package.